pontoon/settings/base.py

```diff
@@ -825,6 +826,10 @@ def account_username(user):
     }
 }
 
+# Google Accounts
+GOOGLE_CLIENT_ID = os.environ.get('GOOGLE_CLIENT_ID', '')
+GOOGLE_SECRET_KEY = os.environ.get('GOOGLE_SECRET_KEY', '')
+
```

Comment:
Nit: Please move this block under the `#GitHub` block (in front of `# All settings related to the AllAuth`).